packages/react/src/context/slash-id-context.tsx

```diff
@@ -234,6 +259,7 @@ export const SlashIDProvider = ({
     if (state === "initial") {
       const slashId = new SlashID({
         oid,
+        ...(environment && { environment }),
```

Comment:
I didn't add any sort of validation since the Core SDK handles that already, I think marking the other properties as `@deprecated` is enough - let me know WDYT @ikovic @jake-slashid

Reply:
@kaspermroz I think the validation part is OK, core SDK will do that. What happens for customers that upgrade to this version without changing to the new format? Will TS complain a lot? Maybe a union of props would be a better idea? To make sure `environment` cannot be used in the combination with old props.

Reply:
We can do that, this will give you a nice TypeScript error

Reply:
Tried that, the problem is that you lose the nice DX with deprecation warning and JS docs. I'd rather leave that as is and have the deprecation warning + runtime error if you pass in the two options